Survey/Sources/Presentation/Modules/Home/Views/SurveyItemView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SurveyItemView.swift
+//  Survey
+//
+//  Created by Khanh on 10/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SurveyItemView: View {
+
+    @State var isShowing = false
+    @State private var fadeInOut = false
+
+    let survey: Survey
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            mainImageSetup()
+            componentsSetup()
+        }
+        .onAppear(perform: {
+            fadeInOut = false
+            withAnimation(Animation.easeInOut(duration: 1.0)) {
+                fadeInOut.toggle()
+            }
+        })
+        .opacity(fadeInOut ? 1.0 : 0.0)
+    }
+
+    private func componentsSetup() -> some View {
```

Comment:
Please take a look for smaller screen, and iPad screen (optional) while you're at it 🙏 
![Simulator Screen Shot - iPhone 8 - 2022-08-15 at 16 27 09](https://user-images.githubusercontent.com/6356137/184611573-b7c5822b-a3e8-4ca4-8d13-9546d6f8c387.png)

Reply:
Thank you for finding out this bug 💯  , I didn't set fame for component views in tabview and padding for whole view in this screen 🥲.  I fixed it at https://github.com/nkhanh44/ic-survey-ios/pull/18/commits/741e27bb29df17a3f373e2c4f4b8fa67ba11a3cd